### PR TITLE
Update FutureRestore GUI to v1.90

### PR DESCRIPTION
**Pull request by CoocooFroggy**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22178)

### CODE REVIEW

File: manifests/c/CoocooFroggy/FutureRestore-GUI/1.90/CoocooFroggy.FutureRestore-GUI.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: "CoocooFroggy.FutureRestore-GUI"
+PackageVersion: "1.90"
+PackageLocale: "en-US"
+Publisher: "CoocooFroggy"
```
Comment (ItzLevvie) [inline manifests/c/CoocooFroggy/FutureRestore-GUI/1.90/CoocooFroggy.FutureRestore-GUI.yaml:4]:
```suggestion
Publisher: "Unknown"
```
Publisher mismatch. You'll have to change this by recompiling your application with `CoocooFroggy` as your publisher.

File: manifests/c/CoocooFroggy/FutureRestore-GUI/1.90/CoocooFroggy.FutureRestore-GUI.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Author: "CoocooFroggy"
+Moniker: futurerestore-gui
+PackageName: "FutureRestore GUI"
+License: "GNU Lesser General Public License v2.1"
+ShortDescription: "A GUI implementation for FutureRestore written in Java."
+Installers: 
+ - Architecture: "x86"
```
Comment (ItzLevvie) [inline manifests/c/CoocooFroggy/FutureRestore-GUI/1.90/CoocooFroggy.FutureRestore-GUI.yaml:11]:
```suggestion
 - Architecture: "x64"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
​

**Comment by CoocooFroggy** [comment]
> #22178 (comment)

I attempted to recompile and reupload with Publisher as CoocooFroggy, so seeing if that works first

**Comment by CoocooFroggy** [comment]
Oops, forgot to update the hash

**Review by OfficialEsco — APPROVED** [review]
image
🥳

**Comment by dabezt31** [comment]
bro what the fuck even happened here LMAOO

**Pull request merged**